libs/utils/utils.js

```diff
@@ -1069,17 +1069,30 @@ const findReplaceableNodes = (area) => {
 };
 
 let placeholderRequest;
+function getPlaceholderPaths(config) {
+  const rootPath = `${config.locale?.contentRoot}/placeholders`;
+  const placeholderPaths = [`${rootPath}.json`];
```

Comment:
we should add some unit testing here

Reply:
I think this func is pretty simple and I'd prefer to not `export` the function?

Reply:
I'm fine, thanks!